test/token/ERC721/ERC721PausableToken.test.js

```diff
@@ -0,0 +1,32 @@
+const { shouldBehaveLikeERC721PausableToken } = require('./ERC721PausableToken.behavior');
+const { shouldBehaveLikeERC721BasicToken } = require('./ERC721BasicToken.behavior');
+
+const BigNumber = web3.BigNumber;
+const ERC721PausableToken = artifacts.require('ERC721PausableTokenMock.sol');
+
+require('chai')
+  .use(require('chai-bignumber')(BigNumber))
+  .should();
+
+contract('ERC721PausableToken', function ([_, owner, recipient, operator, ...otherAccounts]) {
+  beforeEach(async function () {
+    this.token = await ERC721PausableToken.new({ from: owner });
+  });
+
+  context('when token is paused', function () {
+    shouldBehaveLikeERC721PausableToken([owner, recipient, operator]);
```

Comment:
The behavior function should receive special accounts as their own argument, and then an array of all other accounts for the behavior to use as it pleases.

Therefore, this call should look like `shouldBehaveLikeBlahBlah(owner, otherAccounts)` (since `recipient` and `operator` have no special meaning here), and the behavior itself should look like `shouldBehaveLikeBlahBlah(owner, [recipient, operator])`.

(same for the other behavior call :))